### ISSUE TITLE
Show triggers is misspelled in dalstatement.g4

### ISSUE THREAD

**Issue by clbigdata** [apache/shardingsphere#13599]
## Bug Report

I found **show triggers** is misspelled in dalstatement.g4

**Comment by strongduanmu** [comment]
@clbigdata What version is it? Can you provide more information?

**Comment by clbigdata** [comment]
5.0.0-RC1-SNAPSHOT

**Comment by clbigdata** [comment]
At line 237
```
showTrriggers
    : SHOW TRIGGER fromSchema? showFilter?
    ;
```
It should be like this
```
showTriggers
    : SHOW TRIGGERS fromSchema? showFilter?
    ;
```

**Comment by strongduanmu** [comment]
@clbigdata Are you interested in submitting a PR to fix this problem?

**Comment by Swastyy** [comment]
Hi @clbigdata @strongduanmu Can I do this?

**Comment by strongduanmu** [comment]
@Swastyy Of course, I will assign this issue to you.

**Comment by strongduanmu** [comment]
Fixed by pr #13629.
